Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11-slim
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY app/ /app/
+
+
+EXPOSE 8000
+
+CMD ["python", "main.py"]
```

Comment:
The `CMD ["python", "main.py"]` command assumes that there is a `main.py` file in the `/app` directory. Ensure that this file exists and is the correct entry point for your application.